modules/apps/frontend-js/frontend-js-state-web/.npmbundlerrc

```diff
@@ -0,0 +1,3 @@
+{
+    "ignore": []
```

Comment:
I doubt I need this. I think I had it in here from an earlier iteration when I was trying a different approach to loading the JS.